cli/cmd/upgrade.go

```diff
@@ -62,6 +64,10 @@ func (options *upgradeOptions) upgradeOnlyFlagSet() *pflag.FlagSet {
 		&options.manifests, "from-manifests", options.manifests,
 		"Read config from a Linkerd install YAML rather than from Kubernetes",
 	)
+	flags.BoolVar(
+		&options.force, "force", options.force,
+		"Force upgrade operation despite validation errors",
```

Comment:
I wonder if we can add some clarity to the description on which validation is skipped. It seems like this only skips the `ensureIssuerCertWorksWithAllProxies()` validation, even though it sounds like it will skip all validations (including those on algorithm, validity period, trust chain etc.). 

IOW, the following usage of `--force` won't allow me to just override my expired issuer cert, unless it is part of the trust root's chain, right?

```
bin/linkerd upgrade --force --identity-issuer-certificate-file=issuer.crt --identity-issuer-key-file=issuer.key
× Failed to build upgrade configuration: issuer certificate does not work with the provided provided roots: x509: certificate signed by unknown authority (possibly because of "x509: ECDSA verification failure" while trying to verify candidate authority certificate "identity.linkerd.cluster.local")
For more information: https://linkerd.io/2/tasks/rotating_identity_trust_root/
For troubleshooting help, visit: https://linkerd.io/upgrade/#troubleshooting
```